sdcm/tester.py

```diff
@@ -1129,6 +1129,9 @@ def get_setup_details(self):
 
         new_scylla_packages = self.params.get('update_db_packages')
         setup_details['packages_updated'] = True if new_scylla_packages and os.listdir(new_scylla_packages) else False
+        setup_details['cpu_platform'] = 'UNKNOWN'
+        if is_gce:
+            setup_details['cpu_platform'] = self.cs_db_cluster.nodes[0].extra.get('cpuPlatform', 'UNKNOWN')
```

Comment:
why gce only? I think, for ec2 we can use Instance.platform

Reply:
We want to record the cpu platform of gce instance, it might be changed (upgraded) in future.
When we compare performance result, we want to know if the cpu platform is same.

Got from ec2: `The operating system platform, such as Windows. This is not returned for all platform types.`

It seems the platformn of ec2 is the OS. If the fixed instance type is always uses fixed cpu platform, then we don't need to get and record the cpu platform of ec2 instance.